kedro_mlflow/template/project/run.py

```diff
@@ -48,10 +48,8 @@ class ProjectContext(KedroContext):
     project_version = "{{ cookiecutter.kedro_version }}"
     package_name = "{{ cookiecutter.python_package }}"
     hooks = (
-        MlflowNodeHook(flatten_dict_params=False),
-        MlflowPipelineHook(
-            model_name="{{ cookiecutter.python_package }}", conda_env="src/requirements.txt",
-        ),
+        MlflowNodeHook(),
```

Comment:
There's a [new way to define hooks](https://kedro.readthedocs.io/en/stable/07_extend_kedro/04_hooks.html#registering-your-hook-implementations-with-kedro) through `.kedro.yml` in just released 0.16.5 version. For now, I think we should leave it as it is (in this PR) and create a new issue to adjust kedro-mlflow setup to this new way of registering hooks as it simplifies the hook registration process greatly.

Reply:
Yes, 0.16.5 brings a lot of interesting novelties but it is another issue (or rather a couple of other issues).